opm/output/eclipse/Summary.cpp

```diff
@@ -443,6 +464,9 @@ static const std::unordered_map< std::string, ofun > funs = {
     { "FGLRH", div( production_history< Phase::GAS >,
                     sum( production_history< Phase::WATER >,
                          production_history< Phase::OIL > ) ) },
+
+    /* Region properties */
+    { "RPR" , rpr}
```

Comment:
**If** you chose to amend to this patch, a handy tip:

A trailing comma in an init list like this is legal. If you add it here (as in it was in the case of FGLRH) then it won't show up in the diff next time we add a keyword because then we won't have to add a comma.